defaults/main.yml

```diff
@@ -10,8 +10,14 @@ openio_keystone_bind_address: "{{ openio_bind_address \
 
 openio_keystone_repo_managed: false
 openio_keystone_openstack_distribution: queens
-openio_keystone_nodes_group: keystone
 openio_keystone_db_nodes_group: galera
+openio_keystone_memcached_nodes_group: "{%- if 'memcached-keystone' in groups -%}
+  memcached-keystone
+  {%- elif 'memcached' in groups -%}
+  memcached
+  {%- else -%}
+  oioswift
```

Comment:
I don't understand this one

Reply:
I think this is not strictly needed, but is the second fallback, and looks harmless

Reply:
deploy memcached in the follwing group with order preference:
- `memcached_keystone`
- `memcached` if `memcached_keystone` is not defined
- `oioswift` if neither `memcached` or `memcached_keystone` is defined

Reply:
I don't really understand the missunderstanding on this one :) this is very straight forward